Makefile

```diff
@@ -10,6 +12,21 @@ ifeq ($(USE_LEGACY), true)
 dockerfile = -f Dockerfile.legacy
 endif
 
+mdx:
```

Comment:
the `bundles` package and `/bin/print-webpack-config.ts` don't exist in this repo what's this makefile target for or should it be moved into our main repo?

Reply:
Oh, that was jenn's workaround for `npm link` being weird. It would copy the built dist directly into the main app. Might be able to delete that at this point.